en_US/durability/durability_introduction.md

```diff
@@ -0,0 +1,88 @@
+
+Each durable MQTT message is stored on disk exactly once, regardless of the number of persistent sessions subscribing to the matching topic filter and whether those sessions are currently connected or not. This enables an efficient fan-out of messages to the persistent sessions.
+
+### Durable Storage Architecture
+
+The architecture of EMQX’s durable storage is organized into a hierarchical structure comprising shards, generations, and streams.
+
+#### Shard
+
+This level segregates messages by client, storing them in distinct shards determined by the publisher's client ID. The number of shards is determined by `session_persistence.storage.builtin.n_shards` configuration parameter during the initial startup of EMQX.
+
+#### Generation
+
+Within each shard, data is further divided into generations that span specific time frames. 
+New messages are written only into the current generation, while the previous generations are only accessible for reading. EMQX cleans up old messages by deleting old generations. For how long messages are retained is specified by the `session_persistence.message_retention_period` parameter.
+
+Different generations can organize the data in a different manner, according to the *storage layout* specification. Currently only one layout is supported, designed to optimize the throughput of wildcard subscriptions covering very large number of topics, and the throughput of single-topic subscriptions.
+
+Future updates will introduce additional layouts to optimize for the different types of workloads, such as prioritizing low latency over high throughput for certain applications.
+
+#### Stream
+
+Messages in each shard and generation are split into streams. Streams serve as units of message serialization in EMQX. Streams can contain messages from multiple topics. Various storage layouts can employ different strategies for mapping topics into streams.
+
+Persistent sessions interact with this structure by fetching messages in batches from the streams, with the batch size adjustable via the `session_persistence.max_batch_size` parameter. This comprehensive system ensures that EMQX can efficiently handle durable message storage and retrieval.
+
+### Session Persistence Across Cluster
+
+Each node within an EMQX cluster is assigned a unique site ID, which serves as a stable identifier, independent of the Erlang node name (`emqx@...`). Site IDs are persistent, and they are randomly generated at the first startup of the node. This stability is crucial for maintaining the integrity of persistent sessions and messages, especially in scenarios where nodes might undergo name modifications or reconfigurations. 
```

Comment:
> This stability is crucial for maintaining the integrity of persistent sessions and messages, especially in scenarios where nodes might undergo name modifications or reconfigurations.

This sentence sounds as if the user must somehow worry about stability of the site IDs. In fact, as long as users don't delete data directory, site ID shouldn't change (as opposed to regular EMQX node names). So maybe rephrase it in a more informational style (remove "crucial")?